docker/initial-data.json

```diff
@@ -521,7 +521,7 @@
             "motion_ids": [],
             "motion_recommendation_ids": [],
             "workflow_id": 1,
-            "first_state_of_workflow_id": null,
+            "first_state_of_workflow_id": 1,
```

Comment:
I don't know, what your intention is, but a workflow can only have one first state. Therefore, only one state should have the property `first_state_of_workflow_id` set to `1`...

Reply:
You are correct. I already reverted that change.

Reply:
Nevermind if you have just deleted these changes...